src/clr-core/styles/layout.stories.ts

```diff
@@ -450,11 +450,15 @@ export const spacing = () => {
       <h1 cds-heading="1">Spacing</h1>
 
       <h2 cds-heading="2">Padding</h2>
-      <cds-placeholder cds-layout="p-xs"><span>p-xs</span></cds-placeholder>
-      <cds-placeholder cds-layout="p-sm"><span>p-sm</span></cds-placeholder>
-      <cds-placeholder cds-layout="p-md"><span>p-md</span></cds-placeholder>
-      <cds-placeholder cds-layout="p-lg"><span>p-lg</span></cds-placeholder>
-      <cds-placeholder cds-layout="p-xl"><span>p-xl</span></cds-placeholder>
+      <cds-placeholder pad="none"><span>pad="none"</span></cds-placeholder>
+      <cds-placeholder pad="xs"><span>pad="xs"</span></cds-placeholder>
+      <cds-placeholder pad="sm"><span>pad="sm"</span></cds-placeholder>
+      <cds-placeholder pad="md"><span>pad="md"</span></cds-placeholder>
+      <cds-placeholder pad="lg"><span>pad="lg"</span></cds-placeholder>
+      <cds-placeholder pad="xl"><span>pad="xl"</span></cds-placeholder>
+      <cds-placeholder pad="[md,none]"><span>pad="[md,none]"</span></cds-placeholder>
+      <cds-placeholder pad="[xs,xl]"><span>pad="[xs,xl]"</span></cds-placeholder>
+      <cds-placeholder pad="[lg,sm]"><span>pad="[xs,xl]"</span></cds-placeholder>
```

Comment:
For non `cds-layout` elements we will have to prefix all attrs to prevent name collisions. So prob something like `cds-pad`?

Reply:
Makes sense. How about `cds-pad` across the board? The intent was to have everything using the same API